component/rating-component/rating.css

```diff
@@ -0,0 +1,33 @@
+.rating-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.rating-div {
+    float: left;
+    height: 46px;
+    padding: 0 10px;
+}
+.rating-div:not(:checked) > input {
+   display: none;
+}
+.rating-div:not(:checked) > label {
+    float:right;
+    margin:0 3px;
+    font-size: 1.565rem;
+}
+
+.rating-div > input:checked ~ label {
+    color: #ffc700;    
+}
+.rating-div:not(:checked) > label:hover,
+.rating-div:not(:checked) > label:hover ~ label {
+    color: #deb217;  
+}
+.rating-div > input:checked + label:hover,
+.rating-div > input:checked + label:hover ~ label,
+.rating-div > input:checked ~ label:hover,
+.rating-div > input:checked ~ label:hover ~ label,
+.rating-div > label:hover ~ input:checked ~ label {
+    color: gold;
```

Comment:
if it is a component in the UI library then you can use color constant.

Reply:
Yess will do Rahul.